Fedora 42 on Starfive VisionFive2 (riscv64gc) perf-event-open-sys-4.0.0 no bindings in the root

wild does not build successfully on on Fedora 42 Starfive VisionFive 2.

Tried to install wild on Starfive VisionFive 2 with Fedora 42 on it.
```
cargo --version --verbose
cargo 1.92.0-nightly (966f94733 2025-09-16)
release: 1.92.0-nightly
commit-hash: 966f94733bbc94ca51ff9f1e4c49ad250ebbdc50
commit-date: 2025-09-16
host: riscv64gc-unknown-linux-gnu
libgit2: 1.9.1 (sys:0.20.2 vendored)
libcurl: 8.15.0-DEV (sys:0.4.83+curl-8.15.0 vendored ssl:OpenSSL/3.5.2)
ssl: OpenSSL 3.5.2 5 Aug 2025
os: Fedora 42.0.0 [64-bit]
```

Build output:
```
cargo install --locked --bin wild --git https://github.com/davidlattimore/wild.git wild-linker
    Updating git repository `https://github.com/davidlattimore/wild.git`
    Updating git submodule `https://github.com/rui314/mold`
  Installing wild-linker v0.6.0 (https://github.com/davidlattimore/wild.git#2d5cfde2)
    Updating crates.io index
warning: package `slab v0.4.10` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
  Downloaded sharded-vec-writer v0.4.0
  Downloaded zstd v0.13.3
  Downloaded zstd-safe v7.2.4
  Downloaded uuid v1.18.1
  Downloaded sharded-slab v0.1.7
  Downloaded memchr v2.7.5
  Downloaded indexmap v2.11.4
  Downloaded hashbrown v0.16.0
  Downloaded crossbeam-channel v0.5.15
  Downloaded winnow v0.7.13
  Downloaded zlib-rs v0.5.1
  Downloaded symbolic-demangle v12.16.3
  Downloaded blake3 v1.8.2
  Downloaded tracing-subscriber v0.3.20
  Downloaded cc v1.2.29
  Downloaded bumpalo v3.19.0
  Downloaded syn v2.0.104
  Downloaded flate2 v1.1.2
  Downloaded cpp_demangle v0.4.4
  Downloaded object v0.37.3
  Downloaded gimli v0.32.3
  Downloaded tracing-core v0.1.34
  Downloaded serde_derive v1.0.225
  Downloaded serde_core v1.0.225
  Downloaded miniz_oxide v0.8.9
  Downloaded heapless v0.7.17
  Downloaded embedded-io v0.6.1
  Downloaded bytemuck_derive v1.10.1
  Downloaded bytemuck v1.23.2
  Downloaded proc-macro2 v1.0.95
  Downloaded postcard v1.1.3
  Downloaded perf-event-open-sys v4.0.0
  Downloaded msvc-demangler v0.10.1
  Downloaded jobserver v0.1.34
  Downloaded getrandom v0.3.3
  Downloaded bitflags v2.9.4
  Downloaded tracing-attributes v0.1.30
  Downloaded thread_local v1.1.9
  Downloaded thiserror-impl v2.0.12
  Downloaded thiserror v2.0.12
  Downloaded symbolic-common v12.16.3
  Downloaded smallvec v1.15.1
  Downloaded sharded-offset-map v0.2.0
  Downloaded zstd-sys v2.0.15+zstd.1.5.7
  Downloaded libc v0.2.175
  Downloaded serde v1.0.225
  Downloaded rustc-demangle v0.1.25
  Downloaded perf-event v0.4.8
  Downloaded memmap2 v0.9.8
  Downloaded libz-rs-sys v0.5.1
  Downloaded leb128 v0.2.5
  Downloaded glob v0.3.3
  Downloaded foldhash v0.2.0
  Downloaded fallible-iterator v0.3.0
  Downloaded embedded-io v0.4.0
  Downloaded debugid v0.8.0
  Downloaded crossbeam-queue v0.3.12
  Downloaded constant_time_eq v0.3.1
  Downloaded colosseum v0.2.2
  Downloaded colored v3.0.0
  Downloaded cobs v0.3.0
  Downloaded cfg-if v1.0.1
  Downloaded bytesize v2.1.0
  Downloaded bumpalo-herd v0.1.2
  Downloaded anyhow v1.0.100
  Downloaded adler2 v2.0.1
  Downloaded sharded-vec-writer v0.3.0
  Downloaded matchers v0.2.0
  Downloaded hash32 v0.2.1
  Downloaded atomic-take v1.1.0
  Downloaded 70 crates (5.6MiB) in 3.97s
   Compiling libc v0.2.175
   Compiling proc-macro2 v1.0.95
   Compiling unicode-ident v1.0.18
   Compiling shlex v1.3.0
   Compiling cfg-if v1.0.1
   Compiling crossbeam-utils v0.8.21
   Compiling semver v1.0.26
   Compiling quote v1.0.40
   Compiling syn v2.0.104
   Compiling jobserver v0.1.34
   Compiling stable_deref_trait v1.2.0
   Compiling serde_core v1.0.225
   Compiling rustc_version v0.4.1
   Compiling cc v1.2.29
   Compiling pkg-config v0.3.32
   Compiling serde v1.0.225
   Compiling thiserror v2.0.12
   Compiling getrandom v0.3.3
   Compiling heapless v0.7.17
   Compiling byteorder v1.5.0
   Compiling anyhow v1.0.100
   Compiling rayon-core v1.12.1
   Compiling hash32 v0.2.1
   Compiling crossbeam-epoch v0.9.18
   Compiling crossbeam-deque v0.8.6
   Compiling zstd-sys v2.0.15+zstd.1.5.7
   Compiling uuid v1.18.1
   Compiling once_cell v1.21.3
   Compiling regex-syntax v0.8.5
   Compiling allocator-api2 v0.2.21
   Compiling equivalent v1.0.2
   Compiling zstd-safe v7.2.4
   Compiling object v0.37.3
   Compiling foldhash v0.2.0
   Compiling memchr v2.7.5
   Compiling cpp_demangle v0.4.4
   Compiling regex-automata v0.4.9
   Compiling thiserror-impl v2.0.12
   Compiling serde_derive v1.0.225
   Compiling cobs v0.3.0
   Compiling tracing-attributes v0.1.30
   Compiling hashbrown v0.16.0
   Compiling tracing-core v0.1.34
   Compiling postcard v1.1.3
   Compiling debugid v0.8.0
   Compiling blake3 v1.8.2
   Compiling symbolic-demangle v12.16.3
   Compiling memmap2 v0.9.8
   Compiling either v1.15.0
   Compiling zlib-rs v0.5.1
   Compiling bitflags v2.9.4
   Compiling lazy_static v1.5.0
   Compiling adler2 v2.0.1
   Compiling pin-project-lite v0.2.16
   Compiling tracing v0.1.41
   Compiling miniz_oxide v0.8.9
   Compiling sharded-slab v0.1.7
   Compiling msvc-demangler v0.10.1
   Compiling symbolic-common v12.16.3
   Compiling libz-rs-sys v0.5.1
   Compiling indexmap v2.11.4
   Compiling matchers v0.2.0
   Compiling bytemuck_derive v1.10.1
   Compiling perf-event-open-sys v4.0.0
error[E0432]: unresolved import `crate::bindings`
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/functions.rs:15:5
    |
 15 | use crate::bindings;
    |     ^^^^^^^^^^^^^^^ no `bindings` in the root
    |
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:183:9
    |
181 | #[cfg(target_arch = "aarch64")]
    |       ----------------------- the item is gated behind the `aarch64` feature
182 | #[path = "bindings_aarch64.rs"]
183 | pub mod bindings;
    |         ^^^^^^^^
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:187:9
    |
185 | #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
    |          --------------------------------------------- the item is gated here
186 | #[path = "bindings_x86_64.rs"]
187 | pub mod bindings;
    |         ^^^^^^^^

error[E0432]: unresolved imports `crate::bindings`, `crate::bindings`
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/functions.rs:64:16
    |
 64 |     use crate::bindings::{self, perf_event_attr, perf_event_query_bpf};
    |                ^^^^^^^^   ^^^^ no `bindings` in the root
    |                |
    |                could not find `bindings` in the crate root
    |
    = help: consider importing this unresolved item through its public re-export instead:
            crate::functions::bindings
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:183:9
    |
181 | #[cfg(target_arch = "aarch64")]
    |       ----------------------- the item is gated behind the `aarch64` feature
182 | #[path = "bindings_aarch64.rs"]
183 | pub mod bindings;
    |         ^^^^^^^^
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:187:9
    |
185 | #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
    |          --------------------------------------------- the item is gated here
186 | #[path = "bindings_x86_64.rs"]
187 | pub mod bindings;
    |         ^^^^^^^^
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:183:9
    |
181 | #[cfg(target_arch = "aarch64")]
    |       ----------------------- the item is gated behind the `aarch64` feature
182 | #[path = "bindings_aarch64.rs"]
183 | pub mod bindings;
    |         ^^^^^^^^
note: found an item that was configured out
   --> /home/riscv/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/perf-event-open-sys-4.0.0/src/lib.rs:187:9
    |
185 | #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
    |          --------------------------------------------- the item is gated here
186 | #[path = "bindings_x86_64.rs"]
187 | pub mod bindings;
    |         ^^^^^^^^

For more information about this error, try `rustc --explain E0432`.
error: could not compile `perf-event-open-sys` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `wild-linker v0.6.0 (https://github.com/davidlattimore/wild.git#2d5cfde2)`, intermediate artifacts can be found at `/tmp/cargo-installPwFNAT`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/libwild/Cargo.toml
+++ b/libwild/Cargo.toml
@@ -49,7 +49,7 @@ winnow = { workspace = true }
 zstd = { workspace = true }
 glob = "0.3.2"
 
-[target.'cfg(target_os = "linux")'.dependencies]
+[target.'cfg(all(target_os = "linux", any(target_arch = "x86_64", target_arch = "aarch64")))'.dependencies]
 perf-event = { workspace = true }
 
 [dev-dependencies]

